src/styles/core/_nav.scss

```diff
@@ -13,7 +13,7 @@ $nav-height: rem(52px);
 $nav-height-small: rem(48px);
 $nav-padding: rem(22px);
 $nav-padding-small: rem(16px);
-$nav-padding-wide: rem(30px);
+$nav-padding-wide: rem(20px);
```

Comment:
I feel like we could have just synced the design with the normal spacing, and use 22px. Removing these 2px hardly makes a difference, just clutters the code.

Reply:
To simplify, I deleted the nav-padding-wide var and the design seems to be alright, can you check if I introduced any regression with it? 

https://github.com/apple/swift-docc-render/pull/91/commits/575e31a28aaef1ec94c57fa1b82ac54ee08c709d

Reply:
Well yes. Now its 2px off :) But honestly I think that we should just unify the rest to that.

CC @SamLanier 

<img width="1111" alt="image" src="https://user-images.githubusercontent.com/9863944/159860309-9d870ed6-3cc3-4f55-b389-b2703a796f4e.png">